src/main/java/edu/tamu/scholars/middleware/export/service/CsvExporter.java

```diff
@@ -150,4 +165,25 @@ private String serialize(String value) {
             : value;
     }
 
+    private String capitalize(String text) {
+        if (text == null || text.isEmpty()) {
+            return text;
+        }
+        return text.substring(0, 1).toUpperCase() + text.substring(1);
+    }
+
+    private String formatPersonType(String type) {
+        if (type == null || type.isEmpty()) {
+            return type;
+        }
+        String mapped = config.getPersonTypeMapping().get(type);
+        if (!mapped.equals(type)) return mapped;
+
+        String result = Arrays.stream(type.replaceAll("([a-z])([A-Z])", "$1 $2").split(SPACE_STRING))
+                        .map(this::capitalize)
+                        .collect(Collectors.joining(SPACE_STRING));
+
+        return result;
```

Comment:
```suggestion
        return Arrays.stream(type.replaceAll("([a-z])([A-Z])", "$1 $2").split(SPACE_STRING))
            .map(this::capitalize)
            .collect(Collectors.joining(SPACE_STRING));
```

This is the last call in the function.
It is cleanier to just return the result directly.

Also, reduce the excessive spacing.
It should be 4 space tabbing, based on the rest of the file.

I will also note that the `[a-z])([A-Z]` regex is not multi-lingual safe.
It only handles US ASCII characters.

We should do some research to find a multi-lingual friendly way to do word splitting.

Reply:
Will create note to address this soon.